src/spatial/math.js

```diff
@@ -1104,9 +1125,12 @@ Crafty.math.Matrix2D = (function () {
      * Sets the values of this matrix
      *
      * @public
-     * @sign public {Matrix2D} setValues(Matrix2D);
+     * @sign public {Matrix2D} setValues(Matrix2D matrix);
+     * @param {Matrix2D} matrix - A matrix to copy the values from
+     * @returns {Matrix2D} This matrix after copying the values
+     * 
      * @sign public {Matrix2D} setValues(Number, Number, Number, Number, Number, Number);
-     * @param {Matrix2D|Number} a
+     * @param {Number} a
```

Comment:
Yeah, had trouble thinking of how to describe these.  I believe they're supposed to match the descriptions [here](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setTransform), so I'll copy those.